src/root/components/OperationalUpdatePdfDocument/index.tsx

```diff
@@ -485,17 +485,70 @@ function OperationalUpdatePdfDocument(props: Props) {
             </View>
           </View>
         </View>
-        <Text style={pdfStyles.sectionHeading}>
-          {strings.drefExportDescriptionOfTheEvent}
-        </Text>
-        <View style={pdfStyles.subSection}>
-          <Text style={pdfStyles.subSectionHeading}>
-            {strings.drefOperationalUpdateDescriptionOfEventLabel}
-          </Text>
-          <Text style={pdfStyles.text}>
-            {operationalUpdateResponse.change_since_request}
-          </Text>
-        </View>
+        {(
+          operationalUpdateResponse?.has_change_since_request ||
+          operationalUpdateResponse?.event_description ||
+          operationalUpdateResponse?.event_scope ||
+          operationalUpdateResponse?.anticipatory_actions ||
+          operationalUpdateResponse?.images_details.length > 0
+        ) && (
+            <View>
+              <Text style={pdfStyles.sectionHeading}>
+                {strings.drefExportDescriptionOfTheEvent}
+              </Text>
+              {operationalUpdateResponse.images_details?.map((el) => (
+                <View style={pdfStyles.subSection}>
+                  <PDFImage
+                    style={pdfStyles.mapImage}
+                    src={el.file}
+                  />
+                </View>
+              ))}
+              {operationalUpdateResponse?.has_change_since_request && (
+                <View style={pdfStyles.subSection}>
+                  <Text style={pdfStyles.subSectionHeading}>
+                    {strings.drefOperationalUpdateDescriptionOfEventLabel}
+                  </Text>
+                  <Text style={pdfStyles.text}>
+                    {formatBoolean(operationalUpdateResponse.has_change_since_request)}
+                  </Text>
+                </View>
+              )}
+              {operationalUpdateResponse?.event_description && (
+                <View style={pdfStyles.subSection}>
+                  <Text style={pdfStyles.subSectionHeading}>
+                    {isImminentOnset ?
+                      strings.drefExportImminentWhereWhenHow
+                      :
+                      strings.drefExportWhatWhereWhen}
+                  </Text>
```

Comment:
This is a weird formatting choice

Reply:
can you please suggest the formatting type for this case?